Source/Hammer.spoon/init.lua

```diff
@@ -0,0 +1,95 @@
+--- === Hammer ===
```

Comment:
Fair comment - my idea was to use this spoon as an umbrella for other common Hammerspoon configuration or automation tasks. That's why I also included the CLI install, the console toggle keybinding, etc. Since by themselves those are all very little things, I thought putting them together in a single spoon would make sense. I'm willing to go either way.